### ISSUE TITLE
Unable to show thumbnail in File component

### ISSUE THREAD

**Issue by mrcportillo** [formio/formio.js#1041]
Hello everyone I have some forms with a File component, when I attach an image, it is uploaded to the file server but the thumbnail is not shown. I need to send the form or navigate to another page (wizard form) and return to see the thumbnail, that is because the download is not made after uploading the file. I leave some screens and part of my code where I configure the file service.

```
    import { Formio } from '@kelsus/formiojs';
    Formio.setProjectUrl('https://apps******xf.for***s.com');
    Formio.setUser({
      email: 'defau****user@tt.com',
      fullname: 'default formio user',
    });
    Formio.setToken( '***' );

    const formioUrl = 'https://apps******xf.for***s.com/form/5c703**985';
    const fileServiceFormio = new Formio(formioUrl);
    let fileService = {
      uploadFile: (storage, file, fileName, dir, progressCallback, url) => {
        return Formio.providers.storage
          .s3(fileServiceFormio)
          .uploadFile(file, fileName, dir, progressCallback, url);
      },
      downloadFile: file => {
        return Formio.providers.storage
          .s3(fileServiceFormio)
          .downloadFile(file);
      },
    };
    Formio.createForm(document.getElementById('formio'), form, {
      fileService,
    }).....
```

![screen shot 2019-02-25 at 10 27 17 am](https://user-images.githubusercontent.com/3878312/53340715-6041bd80-38e8-11e9-95ea-49fd5d16d99a.png)

![screen shot 2019-02-25 at 10 27 28 am](https://user-images.githubusercontent.com/3878312/53340720-659f0800-38e8-11e9-9694-9facd54df0f3.png)

**Comment by mrcportillo** [comment]
I did this modification on my fork and it works:
![screen shot 2019-02-27 at 4 01 30 pm](https://user-images.githubusercontent.com/3878312/53515562-076d5300-3aa9-11e9-9801-83fa675f52ff.png)

**Comment by travist** [comment]
This is fixed in **3.16.2**
